Logos/Morphisec_Logo.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="f8a78c88-c469-46fa-be85-47fa820f7cd7" xmlns="http://www.w3.org/2000/svg" version="1.1" width="72" height="72" viewBox="0 0 72 72">
```

Comment:
svg code file have invalid attributes. please follow the below guidelines:

-The company logo adhering to the following requirements in the 'Logo' folder
1. Logo needs to be in SVG format and under 5 Kb
2. Ensure raw file of logo does not have any of the following:
a) cls and style formats
b) embedded png formats
c) xmlns:xlink
d)data-name
3. Do not use xlink:href - use inline instead
4. Do not use title tag
5. If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
6. Logo scales well to fit in a 75 px square
7. SVG code file is formatted well for readability

for more detail please follow the link:
https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md

Reply:
Replaced logo into smaller one without xmlns

Reply:
logo is not rendering in frame properly. Please find the screenshot below:
![image](https://user-images.githubusercontent.com/73637076/102897310-ce784c80-448d-11eb-8099-3b83421434ce.png)

I have made changes in the viewbox from {0 0 72 72} to {0 00  150 150} .Please find the updated code in file below shared:
[MorphiescSVGCodeFile.txt](https://github.com/Azure/Azure-Sentinel/files/5730273/MorphiescSVGCodeFile.txt)